src/components/metadata/metadata.js

```diff
@@ -345,6 +347,29 @@ const MetaData = ({
                   </button>
                 </>
               )}
+              {hasJSON && (
+                <>
+                  <div className="pipeline-metadata__preview-json">
+                    <JSONObject
+                      value={JSON.parse(metadata.preview)}
+                      theme={theme}
+                      style={{ background: 'transparent', fontSize: '14px' }}
```

Comment:
is it fine to use inline style or should be fallback to className ?

Reply:
i think we might not be able to use classNames here because of the react-json-view is made - https://www.npmjs.com/package/react-json-view -- it just has a theme and style attribute

Reply:
Yes we checked for `className` attribute in react-json-view but didn't find so Sajid used style.